build_mt.py

```diff
@@ -378,24 +369,20 @@ def main(sim_name, z_start, z_stop, merger_parent, catalog_parent, resume=False,
         print("comoving distance between this and previous snapshot = ", delta_chi)
 
         # read merger trees file names at this and previous snapshot from minified version 
-        fns_this = merger_dir.glob(f'associations_z{z_this:4.3f}.*.asdf.minified')
-        fns_prev = merger_dir.glob(f'associations_z{z_prev:4.3f}.*.asdf.minified')
-        fns_this = list(fns_this)
-        fns_prev = list(fns_prev)
+        # LHG: do we need to support both minified and non-minified separately? I thought all the data was in the minifted format now.
```

Comment:
Do we need to support minified and non-minified?  I thought everything was minified now (and that we had deleted the originals, and renamed the minified to remove "minified" in the filename).